### ISSUE TITLE
Allow configuration to choose level of WCAG conformance (A vs. AA vs. AAA)

### ISSUE THREAD

**Issue by gerardo-rodriguez** [cloudfour/pleasantest#639]
Most teams/projects will likely only want to conform to WCAG Level AA guidelines. This should be the default.

Other tests (such as the 44x44 touch target area which is part of Level AAA) should be opt-in only and not forced upon Pleasantest users.
